src/plugins/navigation/public/top_nav_menu/top_nav_control_item.tsx

```diff
@@ -60,6 +60,7 @@ export function TopNavControlItem(props: TopNavControlData) {
         {props.description}
         {links?.map((linkProps) => (
           <>
+            {' '}
```

Comment:
Was this intentional?

Reply:
Thanks for the attention to detail. Yes. When multiple links are added, this ensures that some space exists between them. It also adds space before the first one.